provisioning/baremetal_pod.go

```diff
@@ -218,6 +238,7 @@ func createContainerMetal3BaremetalOperator(images *Images, config *metal3iov1al
 		ImagePullPolicy: "IfNotPresent",
 		VolumeMounts: []corev1.VolumeMount{
 			ironicCredentialsMount,
+			rpcCredentialsMount,
```

Comment:
baremetal-operator doesn't need RPC creds. It's an interface that is purely internal to Ironic.

Reply:
Derek mentioned in the original PR that Ironic needs the creds to talk to itself?

Reply:
Ironic Conductor, but not BMO.

Reply:
Done